docs/usage/self-hosted-configuration.md

```diff
@@ -242,16 +242,12 @@ Warning: Configuring `logLevel` config option or `--log-level` cli option is dep
 
 ## onboarding
 
-Set this to `false` if (a) you configure Renovate entirely on the bot side (i.e. empty `renovate.json` in repositories) and (b) you wish to run Renovate on every repository the bot has access to, and (c) you wish to skip the onboarding PRs.
```

Comment:
This sentence is long and hard to understand, so I'd like to replace it with a list.
Please check if the list provided is correct.

Reply:
I can't make suggestions on deleted lines so perhaps you can change the current one into a list and then we edit